frappe/model/rename_doc.py

```diff
@@ -492,7 +500,7 @@ def bulk_rename(doctype, rows=None, via_console = False):
 		if len(row) > 1 and row[0] and row[1]:
 			merge = len(row) > 2 and (row[2] == "1" or row[2].lower() == "true")
 			try:
-				if rename_doc(doctype, row[0], row[1], merge=merge):
+				if rename_doc(doctype, row[0], row[1], merge=merge, rebuild_search=False):
```

Comment:
I think we should enqueue rebuild search after all the rows are renamed for bulk_rename.

Reply:
https://github.com/frappe/frappe/blob/429d5178e00572ac74086dc529f9fd126ffd55f9/frappe/model/rename_doc.py#L517

Reply:
Did you see this? Should suffice, I think.

Reply:
Ahh, I didn't expand the code!! This looks good then